lib/rubocop/cop/style/case_equality.rb

```diff
@@ -58,7 +58,7 @@ def const?(node)
         def replacement(lhs, rhs)
           case lhs.type
           when :regexp
-            "#{rhs.source} =~ #{lhs.source}"
+            "#{lhs.source}.match?(#{rhs.source})"
```

Comment:
Before doing this, you need to check if the subsequent code does not refer to the match data (`Regexp.last_match`, `$~`, `$1`, etc.).

Reply:
Sure! I have a idea. `Performance/Regexp` cop detects `Regexp.last_match`, and etc. It is expected that `a === b` will be auto-corrected to `a.match?(b)` in future.
Reelated issue: https://github.com/rubocop-hq/rubocop-performance/issues/152

It currently doesn't support unsafe auto-correction, so it may be better to delegate it to `Perfomance/Regexp` cop.